### ISSUE TITLE
Colon used as separator

### ISSUE THREAD

**Issue by drarmstr** [DiffSK/configobj#83]
Would you consider allowing either the colon or the equal sign as a separator as ConfigParser supports?

**Comment by EliAndrewC** [comment]
I'm not opposed to that in principle, but we'd need to make sure it's backwards-compatible.  For better or for worse, if an existing config file has a line like

```
foo : bar = baz
```

then that will parse to an entry where ``'foo : bar'`` maps to a value of ``baz``.

However, if there was an option you could pass to the ``configobj.ConfigObj`` class to indicate that you wanted to use colons as a separator, that would probably be okay.

(Theoretically we could use colon as a separator if and only if there are no equals signs on a line.  However, I'm dubious about introducing heuristics and guesswork to the parser.)

**Comment by drarmstr** [comment]
Agreed, adding an option with a set of separators would be the safest, cleanest, and definitely has precedence with other interfaces. 

But, I think this change would really help ConfigObj be a compatible replacement for ConfigParser and help adoption.

**Comment by kierun** [comment]
Any chance of this being in the next release?

**Comment by jhermann** [comment]
You can improve chances significantly by a PR. :trollface:

**Comment by kierun** [comment]
You mean like [this](https://github.com/DiffSK/configobj/pull/118)? :trollface:

**Comment by kierun** [comment]
So, what is the best way forward?…
